content/python/concepts/strings/terms/casefold/casefold.md

```diff
@@ -39,3 +39,13 @@ text = "this Should All Be In Lower Case!"
 x = text.casefold()
 print(x)
 ```
+
+## Example 3
+
+Use `casefold()` to set all characters in a string to lower case:
```

Comment:
```suggestion
`casefold()` is used to set all characters in a string to lower case:
```

Changed this to be more declarative rather than suggestive. Nice codebyte usage 👍